subprojects/plugin-use/src/integTest/groovy/org/gradle/plugin/use/VersionInSettingsPluginUseIntegrationTest.groovy

```diff
@@ -0,0 +1,241 @@
+/*
+ * Copyright 2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradle.plugin.use
+
+import org.gradle.integtests.fixtures.AbstractIntegrationSpec
+import org.gradle.test.fixtures.plugin.PluginBuilder
+import org.gradle.test.fixtures.server.http.MavenHttpPluginRepository
+import org.junit.Rule
+
+class VersionInSettingsPluginUseIntegrationTest extends AbstractIntegrationSpec {
```

Comment:
It would be great to also cover DSL usage in Kotlin scripts

Reply:
b6fca7e25d829a40c79faaaf5058c719bf135b2e